config-example.yaml

```diff
@@ -30,7 +30,7 @@ keep_alive_monitor:
 # on your farm performance at the specified time of the day.
 daily_stats:
   enable: true
-  time_of_day: 21
+  time_of_day: "21:00"
```

Comment:
The existing integer value still works, hence may not necessary changing this to 21:00, but I find it more intuitive defining hh:mm